hibernate-core/src/test/java/org/hibernate/orm/test/stateless/events/CollectionListenerInStatelessSessionTest.java

```diff
@@ -97,6 +104,8 @@ class MyPostCollectionRemoveEventListener implements PostCollectionRemoveEventLi
 
 	@Override
 	public void onPostRemoveCollection(PostCollectionRemoveEvent event) {
+		assertThat( event.getAffectedOwnerOrNull() ).isNotNull();
+		assertThat( event.getCollection().getOwner() ).isNotNull();
```

Comment:
We want the owner to be non-null _after_ a remove?

Reply:
that ... idk 🤷🏻 🙂 
wouldn't we want to know to what the collection belonged to before it was removed?
My thinking is that `event.getAffectedOwnerOrNull() == event.getCollection().getOwner()`

Reply:
If you're sure this is pre-existing behavior and we don't need to change it to fix the bug, fine by me.